flutter/lib/src/uvccamera_controller.dart

```diff
@@ -83,9 +87,8 @@ class UvcCameraController extends ValueNotifier<UvcCameraControllerState> {
 
     _isDisposed = true;
 
-    if (_initializeFuture != null) {
-      await _initializeFuture;
```

Comment:
But that's harmless, completed future will do exactly that. What's the benefit?

Reply:
just to be able to check if it's isCompleted, what would ignore Future that already happened, and we would only wait if it's ongoing

Reply:
This is done intentionally, prior to releasing, you should wait for initialization to be completed, if it was started.

Reply:
True, but awaiting a completed future would effectively do the same. It won't wait for anything if that future is completed. There's no need to be explicit about that?